test/SILGen/objc_blocks_bridging.swift

```diff
@@ -154,7 +154,7 @@ class Test: NSObject {
 // CHECK:         return [[RESULT]]
 
 func clearDraggingItemImageComponentsProvider(_ x: NSDraggingItem) {
-  x.imageComponentsProvider = {}
+  x.imageComponentsProvider = { [] }
```

Comment:
Is this a language change? Does it need to be guarded under `-swift-version 5`?

Reply:
I rewrote the commit message just before pushing and accidentally deleted the sentence about the updated test cases being ones where data flow diagnostics emit an error.

Reply:
I didn't comment about it because I tried second modified test-case and it did produce an error without `return 0`